New package: CoupledElectricMagneticDipoles v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: CoupledElectricMagneticDipoles
- Repository: https://github.com/augustinmuster/CoupledElectricMagneticDipoles.jl
- Created by: @augustinmuster
- Version: v0.1.0
- Commit: b9a64b7514120d54590b407aa453d443c52e2234
- Reviewed by: @augustinmuster
- Reference: https://github.com/augustinmuster/CoupledElectricMagneticDipoles.jl/issues/10#issuecomment-1953841389
- Description: Implementation of DDA and CEMD method in julia
<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a835987058b87a423c833e6c7d722027db5d4db5c32c47e2f28e7481a054641c1b95a51fbeede5d26cba90bc0bc1c2ad566d651e7fe5d8fed4fadbb46e76ee8afea331f3dc59d4b5479294a330a6fc3ea8356a48dc8d79c270e9272841a2527efce7c6c317a76266c46be8f8c06609ee3f409d259941043f5681a52797dd6eb28cce41c0433c8dd3984fe71c26d8bc144443d3 -->

augustinmuster commented:
We have to fix bugs for this first version. Please don't merge the pull request until we pushed these changes on Friday 16th February 2024 .

JuliaRegistrator committed:
New package: CoupledElectricMagneticDipoles v0.1.0

UUID: f3c9a48b-9994-415a-a861-0bea7d656b27
Repo: https://github.com/augustinmuster/CoupledElectricMagneticDipoles.jl.git
Tree: 2fddb8f072fb3fbd53eef222de02e974838a775c

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

GunnarFarneback commented:
[noblock]

If you want this to be merged, you need to edit your comment to include `[noblock]`.
